packages/fxa-payments-server/src/index.scss

```diff
@@ -56,6 +45,10 @@ h3.billing-title {
       margin: 0 -20px; // match padding of #main-content
     }
   }
+
+  @include respond-to('big') {
```

Comment:
Old breakpoint 👎 Let's add payment breakpoint updates to the cleanup ticket we discussed?